homeassistant/components/hunterdouglas_powerview/scene.py

```diff
@@ -35,25 +35,14 @@ async def async_setup_entry(
 class PowerViewScene(HDEntity, Scene):
```

Comment:
This file should be in `.coveragerc` (this integration is really old) but thats an existing problem not specific to this PR

Reply:
Will do that when I'm back at my pc

Reply:
Sorry, was too quick.  Better to do in a separate PR anyways.